app/client/rest/emojis.ts

```diff
@@ -15,7 +15,7 @@ export interface ClientEmojisMix {
     getCustomEmojiByName: (name: string) => Promise<CustomEmoji>;
     getCustomEmojis: (page?: number, perPage?: number, sort?: string) => Promise<CustomEmoji[]>;
     deleteCustomEmoji: (emojiId: string) => Promise<any>;
-    getSystemEmojiImageUrl: (filename: string) => string;
+    getStandardEmojiImageUrl: (filename: string) => string;
```

Comment:
lol thanks for renaming.. but this is not needed anymore, right ?

Reply:
no, I removed another one, I leave the type as it defines how the standard emojis look like in case you need that in the future